internal/configuration/command.go

```diff
@@ -0,0 +1,112 @@
+package configuration
+
+import (
+	"fmt"
+	"reflect"
+	"runtime"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/confluentinc/cli/v3/internal/update"
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/config"
+	"github.com/confluentinc/cli/v3/pkg/types"
+)
+
+const fieldNotConfigurableError = `configuration field "%s" does not exist or is not configurable`
+
+type command struct {
+	*pcmd.CLICommand
+	cfg *config.Config
+}
+
+type fieldInfo struct {
+	kind     reflect.Kind
+	name     string
+	readOnly bool
+}
+
+type fieldOut struct {
+	Name     string `human:"Name" serialized:"name"`
+	Value    string `human:"Value" serialized:"value"`
+	ReadOnly bool   `human:"Read-Only" serialized:"read_only"`
+}
+
+func New(cfg *config.Config, prerunner pcmd.PreRunner) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "configuration",
+		Aliases: []string{"config"},
+		Short:   "Configure the Confluent CLI.",
+	}
+
+	c := &command{
+		CLICommand: pcmd.NewAnonymousCLICommand(cmd, prerunner),
+		cfg:        cfg,
+	}
+
+	cmd.AddCommand(c.newDescribeCommand())
+	cmd.AddCommand(c.newListCommand())
+	cmd.AddCommand(c.newUpdateCommand())
+
+	return cmd
+}
+
+func getWhitelist(cfg *config.Config) map[string]*fieldInfo {
+	whitelist := make(map[string]*fieldInfo)
+	t := reflect.TypeOf(*cfg)
+	for i := 0; i < t.NumField(); i++ {
+		field := t.Field(i)
+		// currently only boolean fields are part of this whitelist, but this may change in the future
+		if kind := field.Type.Kind(); kind == reflect.Bool {
```

Comment:
What if we add a boolean field that we don't want to show up here? Can we make an actual whitelist slice of fields that we want to display?

Reply:
If we add a boolean field to the `config` struct, it will fail the `getWhitelist` unit test, so if we don't want it to be added we would have to modify this function. 

Yeah, I guess we could make an actual whitelist along side the `config` struct, but we can't avoid having to add `disable_plugins_once` to it at runtime.

Reply:
> we can't avoid having to add disable_plugins_once to it at runtime

Why not?

Reply:
We only want to add this field if the CLI is on Windows. I think I mentioned this before, but if a Unix/mac user sees this field, they may think setting it will actually do something, but it won't. 

One alternative would be to include it in the whitelist, but make it read-only for non-Windows users.

Reply:
What if we just define a slice of whitelisted configurations, and only add that field if `runtime.GOOS == "windows"`?

Reply:
My wording was probably confusing earlier. I meant that `disable_plugins_once` cannot be included in the map literal's declaration, and instead must be added at runtime if `runtime.GOOS == "windows"`.

What u said is exactly what I'm doing haha. Sorry for the misunderstanding.